simulation_parameters/microbe_stage/compounds.json

```diff
@@ -209,7 +209,7 @@
     "Volume": 1,
     "IsCloud": false,
     "IsAlwaysUseful": false,
-    "IsEnvironmental": true,
+    "IsEnvironmental": false,
```

Comment:
This will break photosynthesis entirely... as the process system won't anymore pick up the sunlight from the environment.